api/source/controllers/Asset.js

```diff
@@ -597,3 +597,116 @@ module.exports.updateAsset = async function updateAsset (req, res, next) {
   }
 }
 
+
+async function getAssetIdAndCheckPermission(request) {
+  const elevate = request.query.elevate
```

Comment:
`elevate` can be removed here or set to `undefined`

Reply:
set to false